lasagne/nonlinearities.py

```diff
@@ -286,6 +286,71 @@ def softplus(x):
     return theano.tensor.nnet.softplus(x)
 
 
+# Clipped Activation
+class ClippedActivation(object):
+    """Clipped Activation activation function
+    :math:`\\varphi(x,c,a) = \\min(a(x),c)`
+
+    This function clips the activation function. Usefull for non-saturating
```

Comment:
And maybe it should state more directly that it modifies an existing activation function:
```
Wraps an existing activation function to clip its output range. Useful for non-saturating
functions such as the linear rectifier to avoid exploding activations [1]_.
```